app/src/main/java/net/skyscanner/backpack/demo/compose/FloatingNotificationStory.kt

```diff
@@ -53,46 +53,48 @@ fun FloatingNotificationStory(modifier: Modifier = Modifier) {
         val text = stringResource(R.string.floating_notification_saved)
         val cta = stringResource(R.string.floating_notification_view)
 
-        Column(
-            modifier = Modifier.padding(BpkSpacing.Base),
-            verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
-        ) {
+        val automationMode = LocalAutomationMode.current
```

Comment:
this makes sure the buttons don't appear on the story screenshot which is confusing